docs/no_numerical_exp_report.tex

```diff
@@ -72,17 +76,86 @@ \section{Method}
+predicts future values with a linear curve fit of the latest
+datasets, and the Moving Average method does so by fitting the 
+errors \cite{stine_introduction_2011}.
+
+The model is referred to as ARMA(p,q), where the p and q represent 
+the order (number of previous time step values fitted) of the autoregressive,
+and the moving average parts,
+respectively. 
+The \gls{ARMA} model can be represented by the following equation:
+
+\begin{align}
+    X_i&=c + \epsilon_t + {\mathlarger{\sum}}_{i=1}^{p}\theta_i \cdot X_{t-i}+
+     {\mathlarger{\sum}}_{j=1}^{q}\phi_j\cdot\epsilon_{t-j}\\
+    X_i&= \mbox{next predicted value of the time series}\\
+    c&= \mbox{adjustable constant}\\
+    \theta_i&=i^{th} \mbox{parameter}\\
+    \phi_i&=j^{th} \mbox{parameter of the moving average system}\\
+    \epsilon &= \mbox{white noise}
+\end{align}
+
+\gls{ARMA} is applied to a `well behaved' set of time series data,
+where there is little volatility. This makes \gls{ARMA} a suitable
+candidate for demand prediction in the case of slowly changing power demand and 
+corresponding reactor deployment.
+
+
+\subsection{Autoregressive Conditional Heteroskedastic (ARCH)}
+The \gls{ARCH} model is similar to the \gls{ARMA} model, except that
+it uses previous variance terms to calculate current error terms, instead
+of the value itself. This allows the model to be used in highly volatile 
+time series (e.g. prediction of inflation or stock prices over time 
+\cite{bollerslev_generalized_1986}).
+The \gls{ARCH} model can be represented by the following equation:
+
+\begin{align}
+    \epsilon_t^2 &= \alpha_0+{\mathlarger{\sum}}_{i=1}^{q}\alpha_i\cdot\epsilon_{t-1}^2\\
+    \alpha_i&= i^{th} \mbox{parameter of the conditional heteroskedastic model}
+\end{align}
+
+A comprehensive fuel cycle simulator must have predictive capabilities which 
+can deploy fuel cycle support facilities intelligently even in the face of 
+volatile dynamics. Such volatility may arise during transitions between 
+technologies, from upsets related to unexpected facility shutdowns, due to the variability 
+of an increasingly renewable electric grid, or other nonlinear economic 
+drivers.  In these cases, the \gls{ARCH} model is more generically applicable 
+and should therefore be preferred over the \gls{ARMA} 
+model.
+
+\subsection{Time Step Execution in \Cyclus}
+As a reference, the time step execution for \Cyclus is illustrated in figure \ref{diag:time}.
+
+At \texttt{Tock}, the algorithm calculates the demand and supply for the next timestep, and calculates
+the difference. If the difference is bigger than the capacity of one facility, it
+schedules to deploy a new facility, or a decommissioning of an existing facility.
+The decommissioning takes place in the \texttt{Decommission} phase of \Cyclus, and
+the construction takes place in the \texttt{Build} phase of \Cyclus. Thus, all the
+adjustment of facilities occur prior to the next DRE.
```

Comment:
might wanna spell dynamic resource exchange out, because i don't think it is mentioned before this.